lib/spack/spack/cmd/buildcache.py

```diff
@@ -398,7 +402,7 @@ def _createtarball(env, spec_yaml=None, packages=None, add_spec=True,
     tty.debug('writing tarballs to %s/build_cache' % outdir)
 
     for spec in specs:
-        tty.debug('creating binary cache file for package %s ' % spec.format())
+        tty.info('creating binary cache file for package %s ' % spec.format())
```

Comment:
Why not change to string format here to be consistent with change at line 261?

Reply:
Actually this is unrelated to the patch itself, let me just undo it, it's rather my own debugging statements that I didn't clean up.